pkg/api/types.go

```diff
@@ -14,6 +15,49 @@ const (
 	StatusError        OpenSCAPStatus = "Error"
 )
 
+// ScanResult represents the compacted result of all scans performed on the image
+type ScanResult struct {
+	// ImageName is a full pull spec of the input image
+	ImageName string `json:"imageName"`
+	// ImageIUD is a SHA256 identifier of the scanned image
+	ImageID string `json:"imageID"`
+	// Results contains compacted results of various scans performed on the image.
+	// Empty results means no problems were found with the given image.
+	Results []Result `json:"results,omitempty"`
```

Comment:
Do we need to start thinking about a `Spec` and `Status` separation here for the future?  Use case: where can report that the image-inspector itself failed in a robust way?  Maybe that's just a pod log for now, but it's something I've been considering.